scss/_patterns_tooltips.scss

```diff
@@ -71,6 +81,11 @@ $triangle-height: $sp-unit;
         -webkit-transform: translateX(-50%); // stylelint-disable-line property-no-vendor-prefix
         transform: translateX(-50%);
       }
+
+      &::after {
+        border-radius: 50% 50% 0% 100% / 100% 100% 0% 0%;
```

Comment:
Would using direct props such as `border-top-right-radius` etc help and reduce the number of overrides we do here in all the variants?
Would be good to only modify borders that need to be modified, not redefine everything.